tests/tests_app/cli/test_connect_data.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from unittest.mock import MagicMock
+
+import pytest
+from lightning_cloud.openapi import ProjectIdDataConnectionsBody, V1ListMembershipsResponse, V1Membership
+
+from lightning.app.cli.connect import data
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="not supported on windows yet")
+def test_connect_data(monkeypatch):
```

Comment:
This test doesn't cover several code paths in the connect_data command.

Reply:
Done !